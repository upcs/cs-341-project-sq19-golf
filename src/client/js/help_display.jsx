import React, { Component } from 'react';
import {Link} from 'react-router-dom';
import '../css/styles.css';

//Options for About Page and Quit to Initial Page
export class HelpContainer extends Component {
	constructor(props) {
		super(props);

		let defaultOnClick = () => alert('UPSchedule is a convenient schedule planner created by students, for students.');
		this.state = {
			onClick: this.props.onClick || defaultOnClick
		};
	};

	render() {
		return (
		<section id="main">
			<div className="textContainer">
				<div className="contentsTable">
					<b id="tableTitle">Help Directory</b>
					<ul>
					<li><a href="#q1">Question 1</a></li>
					</ul>
				</div>
				<br />
				<div className="helpSet" id="h1">
					<a id="q1">Question</a>
					<p className="helpContent" id="a1">Answer</p>
				</div>

			</div>
		</section>
		);
	}
}
