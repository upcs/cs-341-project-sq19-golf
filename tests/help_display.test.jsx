import React from 'react';
import { shallow } from 'enzyme';
import Sinon from 'sinon';

import {HelpContainer} from '../src/client/js/help_display.jsx';

describe('helpContainer', () => {
  test('Should render correctly', () => {
    const helpContainer = shallow(<HelpContainer/>);
    expect(helpContainer).toMatchSnapshot();
  });

});