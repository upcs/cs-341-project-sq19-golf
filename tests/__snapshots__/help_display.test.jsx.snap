// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helpContainer Should render correctly 1`] = `
<section
  id="main"
>
  <div
    className="textContainer"
  >
    <div
      className="contentsTable"
    >
      <b
        id="tableTitle"
      >
        Help Directory
      </b>
      <ul>
        <li>
          <a
            href="#q1"
          >
            Question 1
          </a>
        </li>
      </ul>
    </div>
    <br />
    <div
      className="helpSet"
      id="h1"
    >
      <a
        id="q1"
      >
        Question
      </a>
      <p
        className="helpContent"
        id="a1"
      >
        Answer
      </p>
    </div>
  </div>
</section>
`;
